evennia/server/evennia_launcher.py

```diff
@@ -1480,14 +1478,118 @@ def create_superuser():
     password = environ.get("EVENNIA_SUPERUSER_PASSWORD")
 
     if (username is not None) and (password is not None) and len(password) > 0:
-
         superuser = AccountDB.objects.create_superuser(username, email, password)
         superuser.save()
     else:
         django.core.management.call_command("createsuperuser", interactive=True)
 
     return True
 
+def create_user(account_name):
```

Comment:
`Account.create(username=..., password=..., email=..., ...)`, is intended to centralize account creation from different places around Evennia, and should already do all the validation that you are doing manually here. Is there a reason this was not used?